packages/ui-spinner/src/Spinner/index.js

```diff
@@ -22,7 +22,7 @@
  * SOFTWARE.
  */
 /** @jsx jsx */
-import React, { useMemo } from 'react'
+import { useMemo } from 'react'
```

Comment:
Does the component render correctly even without `React` being in context?

Reply:
It makes no visible difference and the linter thinks it's unnecessary